src/requirements/data/checkers-common.ts

```diff
@@ -38,9 +39,19 @@ export const courseMatchesCodeOptions = (course: Course, codeOptions: readonly s
  * @returns if the course satisfies FWS requirement.
  */
 export const courseIsFWS = (course: Course): boolean =>
+  course.crseId === FWS_COURSE_ID ||
   course.titleLong.includes('FWS:') ||
   (course.catalogSatisfiesReq?.includes('First-Year Writing Seminar') ?? false);
 
+/**
+ * Detects special (synthetic) courses, as defined in requirement-json-generator.ts
+ *
+ * @param course course object with useful information retrived from Cornell courses API.
+ * @returns if the course is a special course
+ */
+export const courseIsSpecial = (course: Course): boolean =>
```

Comment:
Can you more explicitly state what a "special course" is?

Reply:
It's explicitly defined in `requirement-json-generator.ts`. I'm hesitant to include something like "credits-only or FWS equivalent course" here in case we modify the special courses in the future, because then we'd need to change the documentation in multiple places. Maybe there should be another location where it's documented?
![image](https://user-images.githubusercontent.com/34158284/113437973-a619bb80-93b5-11eb-8f10-071037aa84d7.png)

Reply:
I see your point. Since you did indicate that specialCourses are defined in requirement-json-generator.ts in the spec, I think this should be ok for now.